packages/ui-react/src/components/VideoMetaData/VideoMetaData.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import styles from './VideoMetaData.module.scss';
+
+type Props = {
+  attributes: unknown[];
```

Comment:
```suggestion
  attributes: string[];
```

Reply:
I did this intentionally, because the array that is being build is not typecasted correctly. The array elements can be `null`, `number` or `string`. eg. Year is a `number`

If you think we should be more explicit here, then I will change it in all the formatter/array creation utilities. I'm awaiting your response before I do that.

Reply:
I changed it ;) Let me know what you think

Reply:
Yes, I would have wrapped it in `String(playlist.genre)` to be a bit more precise, but this also works